### PR TITLE
Master

### PR DESCRIPTION
**Pull request by christinasund**
Commission Junction - Created more defenses around data type using a pre baked utility
function

### CODE REVIEW

**Comment by EXPEanlagnada** [comment]
Hi Christina, there is a conflict with your fork.  Please merge the latest with upstream.

**Comment by wmoussa** [comment]
Hi Christina, I joined the OMG team couple of months ago. Nice meeting you over Github 👍 Could you please provide a more descriptive PR title? It will help me in my ramping up process. thanks

File: src/library/simplified/tag/criteo.js [hunk 1]
```diff
@@ -0,0 +1,472 @@
+//~~tv:20010.20140827
```
Comment (wmoussa) [inline src/library/simplified/tag/criteo.js:1]: why I cannot see any diff? Was it because that file was never pushed to github originally?

Comment (christinasund) [reply src/library/simplified/tag/criteo.js:1]: Nice to meet you! I believe so, yes. I just added three files so there would be nothing to compare.

**Comment by EXPEanlagnada** [comment]
Still waiting on the conflict resolution on this.

**Pull request merged**